src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -431,22 +443,13 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
             }
             submitLabel={t('Save Rule')}
           >
-            <TriggersChart
-              api={this.api}
-              organization={organization}
-              projects={this.state.projects}
-              triggers={triggers}
-              query={queryAndAlwaysErrorEvents}
-              aggregation={aggregation}
-              timeWindow={timeWindow}
-            />
-
             <RuleConditionsForm
               api={this.api}
               projectSlug={params.projectId}
               organization={organization}
               disabled={!hasAccess}
               onFilterUpdate={this.handleFilterUpdate}
+              thresholdChart={chart}
```

Comment:
Would it be better to move the chart component itself into the form instead of having it as a prop?

Reply:
I mostly just didn't want to have to prop drill the `TriggerChart` props in. But maybe that's okay. What do you think @billyvg ?